src/components/bulk/Ansatte.tsx

```diff
@@ -40,8 +44,9 @@ const Ansatte: React.FC = () => {
     if (!innsendteAnsatte.find((ansatt: Ansatt) => !ansatt.referenceNumber)) {
       setAnsatte([byggAnsatt()])
       setFeil([])
-      history.push('/kvitteringBulk')
+
     }
+    history.push('/bulk/kvittering')
```

Comment:
Det er mulig vi burde hatt pattene som konstanter i en egen fil. For å unngå skrivefeil.

Reply:
Tok deg på ordet og lagde filen Linker.ts med alle pathene i en enum klasse.